pyblish_nuke/lib.py

```diff
@@ -7,20 +7,27 @@
 
 # Host libraries
 import nuke
+import nukescripts
 
 # Local libraries
 from . import plugins
 
+try:
```

Comment:
No need to try/except, Nuke is guaranteed to have at least one binding available.